app/script/conversation/EventMapper.js

```diff
@@ -365,10 +366,11 @@ z.conversation.EventMapper = class EventMapper {
    * Maps JSON data of local decrypt errors to message entity
    *
    * @private
-   * @param {Object} error_code - Error data received as JSON
+   * @param {Object} error - Error data received as JSON
    * @returns {DecryptErrorMessage} Decrypt error message entity
    */
-  _map_event_unable_to_decrypt({error_code}) {
+  _map_event_unable_to_decrypt(error) {
+    const {error_code: error_code} = error;
```

Comment:
Right. I did it because originally I needed type as error_type as well but now it's not the case anymore